docker/README.md

```diff
@@ -0,0 +1,79 @@
+# Using Docker to run garage
+
+Currently there are three types of garage images:
+  - ci: used for the continuous integration of garage in TravisCI and not
+    intended for final users.
```

Comment:
what is a final user?

Reply:
The user that only wants to test/deploy experiments with garage, sorry if the term is misleading. I will remove this section of text anyway.